static/app/views/releases/detail/commitsAndFiles/repositorySwitcher.tsx

```diff
@@ -15,24 +15,7 @@ type Props = {
   activeRepository?: Repository;
 };
 
-type State = {
-  dropdownButtonWidth?: number;
-};
-
-class RepositorySwitcher extends PureComponent<Props, State> {
-  state: State = {};
-
-  componentDidMount() {
-    this.setButtonDropDownWidth();
-  }
-  setButtonDropDownWidth() {
-    const dropdownButtonWidth = this.dropdownButton?.current?.offsetWidth;
-    if (dropdownButtonWidth) {
-      this.setState({dropdownButtonWidth});
-    }
-  }
-
-  dropdownButton = createRef<HTMLButtonElement>();
```

Comment:
`CompactSelect` already does the work to make sure that the dropdown menu's width is always larger or equal to the trigger button's width ([see here](https://github.com/getsentry/sentry/blob/master/static/app/components/forms/compactSelect.tsx#L224-L244)), so we can just remove this.